### PR TITLE
More rubocop fixes

### PR DESCRIPTION
**Pull request by gmcgibbon**
Fixes `Style/FrozenStringLiteralComment` and `Metrics/LineLength` violations.

### CODE REVIEW

File: identity_cache.gemspec [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+    "Harry Brundage",
+    "Dylan Thacker-Smith",
+    "Tobias Lutke",
+    "Arthur Neves",
+    "Francis Bogsanyi",
+  ]
   gem.email         = ["gems@shopify.com"]
-  gem.description   = %q{Opt in read through ActiveRecord caching.}
-  gem.summary       = %q{IdentityCache lets you specify how you want to cache your model objects, at the model level, and adds a number of convenience methods for accessing those objects through the cache. Memcached is used as the backend cache store, and the database is only hit when a copy of the object cannot be found in Memcached.}
+  gem.description   = "Opt in read through ActiveRecord caching."
```
Comment (rafaelfranca) [inline identity_cache.gemspec]:
```suggestion
  gem.description   = "Opt-in, read through, Active Record caching."
```

File: performance/cache_runner.rb [hunk 2]
```diff
@@ -13,7 +14,8 @@
 IdentityCache.logger = Logger.new(nil)
 CacheConnection.setup
 
-if ActiveRecord.gem_version < Gem::Version.new('5') && ActiveRecord::Base.respond_to?(:raise_in_transactional_callbacks=)
+if ActiveRecord.gem_version < Gem::Version.new('5') &&
```
Comment (rafaelfranca) [inline performance/cache_runner.rb]: Do we still support Rails 4?

File: test/test_helper.rb [hunk 3]
```diff
@@ -32,7 +33,8 @@ class IdentityCache::TestCase < Minitest::Test
   attr_reader :backend
 
   def setup
-    if ActiveRecord.gem_version < Gem::Version.new('5') && ActiveRecord::Base.respond_to?(:raise_in_transactional_callbacks=)
+    if ActiveRecord.gem_version < Gem::Version.new('5') &&
```
Comment (rafaelfranca) [inline test/test_helper.rb]: Same about supporting Rails 4

Comment (gmcgibbon) [reply test/test_helper.rb]: 🤦‍♂ We don't, I'll remove those `if`s

**Pull request merged**